projects/robots/husarion/rosbot/controllers/rosbot/rosbot.c

```diff
@@ -73,10 +73,10 @@ int main(int argc, char *argv[]) {
   wb_motor_set_velocity(rear_right_motor, 0.0);
 
   /* get a handler to the position sensors and enable them. */
-  front_left_position_sensor = wb_robot_get_device("front left wheel motor sensor");
-  front_right_position_sensor = wb_robot_get_device("front right wheel motor sensor");
-  rear_left_position_sensor = wb_robot_get_device("rear left wheel motor sensor");
-  rear_right_position_sensor = wb_robot_get_device("rear right wheel motor sensor");
+  front_left_position_sensor = wb_robot_get_device("fl_range");
+  front_right_position_sensor = wb_robot_get_device("fr_range");
```

Comment:
Any reason for the name change? From the new name they might be misjudged for range finders where in fact they are position sensors. Also keep in mind that sensors names will show up in the webots.cloud page (https://webots.cloud/run?version=R2023b&url=https://github.com/cyberbotics/webots/blob/develop/projects/robots/husarion/rosbot/protos/Rosbot.proto&type=undefined) in the "Devices" tab, as such complete names are preferable.

Reply:
Ohhh... Your right it shouldn't be changed. I mistook a wheel position sensor with a distance sensor.